Assets/Scripts/PlayerController.cs

```diff
@@ -56,6 +57,7 @@ void Start()
         targetObject = null;
         bp = GameObject.Find("UI_Canvas").GetComponents<ButtonPromptDisplay>()[0];
         bp2 = GameObject.Find("UI_Canvas").GetComponents<ButtonPromptDisplay>()[1];
+        interactionIcon = GameObject.Find("UI_Canvas").GetComponents<InteractionIcon>()[0];
```

Comment:
Create a string in Globals for "UI_Canvas" and replace this & other instances. Having raw strings in script is against the style convention we're following